### PR TITLE
*: support ssl mysql sink

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve? 

Support sending changes to MySQL sink with SSL enabled.

### Check List 

Tests 

 - Integration test

Code changes

 - Has exported function/method change

Side effects

 - Increased code complexity

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Release note

* Support sending changes to MySQL sink with SSL enabled.

### CODE REVIEW

File: cdc/sink/sink.go [hunk 1]
```diff
@@ -79,7 +65,7 @@ func NewSink(ctx context.Context, sinkURIStr string, filter *filter.Filter, conf
 	case "blackhole":
 		return newBlackHoleSink(opts), nil
 	case "mysql", "tidb":
```
Comment (leoppro) [inline cdc/sink/sink.go:67]:
```suggestion
	case "mysql", "tidb","mysql+ssl", "tidb+ssl":
```
see: https://github.com/pingcap/ticdc/pull/734/files#diff-d6e2759c88c65cddfd8d333dae3e5b94R85

**Comment by amyangfei** [comment]
Seems some integration tests are always failed

File: tests/_utils/run_sql [hunk 2]
```diff
@@ -2,11 +2,17 @@
...
 # parameter 2: database host
 # parameter 3: database port
+# parameter 4: other mysql client settings
 
 set -e
 
-host=${2:-127.0.0.1}
-port=${3:-4000}
+sql=${1}
+shift
```
Comment (amyangfei) [inline tests/_utils/run_sql:10]: The failure test case may be caused by the shift operation.
some `run_sql sql_statement` doesn't provide `host` and `port`

**Pull request merged**